mautic/mautic
# Fix company contacts list

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
If company has not assigned contacts, it show wrong contacts.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contacts
3. Create companies
4. Check detail of company without contact
5. In some cases you see contacts, even If we do not expect it
6. This PR fixed it and added note about no contacts assigned to company

![image](https://user-images.githubusercontent.com/462477/120649022-959cc480-c47c-11eb-86f2-98c0f37e2a64.png)
